lib/components/Buttons/styles.ts

```diff
@@ -0,0 +1,24 @@
+import { css } from "@emotion/css";
+
+export const button_primary = css`
+  box-sizing: border-box;
+  margin: 0px;
+  min-width: 0px;
+  appearance: none;
+  display: inline-block;
+  text-align: center;
+  line-height: inherit;
+  text-decoration: none;
+  padding: 10px 18px;
```

Comment:
padding, margin, borders, widths and heights should all use `rem` so that they scale up equally as the user increases their device font size

Reply:
done